src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
+/* eslint-disable max-len */
 import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie: Movie) => (
```

Comment:
You could write just movie:
`{movies.map(movie => (`

Reply:
**Fixed**